index.html

```diff
@@ -0,0 +1,188 @@
+                                class="quick-link-icon"
+                            />
+                        </li>
+                        <li>
+                            <a href="https://quickbooks.intuit.com/">
+                                quickbooks</a
+                            >
+                        </li>
+                        <li>
+                            <img
+                                src="./images/turbotax-logo.svg"
+                                alt=""
+                                class="quick-link-icon"
+                            />
+                        </li>
+                        <li>
+                            <a href="https://turbotax.intuit.com/"> turbotax</a>
+                        </li>
+                    </ul>
+                </div>
+
+                <section class="sign-up-wrapper sign-up">
+                    <div class="sign-up-area">
+                        <header class="sign-up-header-container form-padding">
+                            <h4 class="sign-up-header form-padding">
+                                Create an Intuit account
+                            </h4>
+                            <span class="sign-up-sub-header"
+                                >One account for everything Intuit, including
+                                Mint. </span
+                            ><a
+                                href="https://accounts-help.lc.intuit.com/questions/1582580-creating-an-account"
+                                >Learn more</a
+                            >
+                            <span class="sign-up-sub-header"
+                                >Already have an account?
+                                <a href="https://accounts.intuit.com/index.html"
+                                    >Sign In</a
+                                ></span
+                            >
+                        </header>
+                        <form
+                            action="/login"
+                            method="POST"
+                            class="sign-up-form form-padding"
+                        >
+                            <label class="label-padding"> Email address</label>
+                            <input class="input-style" type="email" />
+
+                            <label class="label-padding">
+                                Phone (recommended)</label
+                            >
+                            <input class="input-style" type="tel" />
+
+                            <span class="phone-input-info"
+                                >Standard call, messaging or data rates may
+                                apply.</span
+                            >
+                            <label class="label-padding"> Password</label>
+
+                            <input
+                                class="input-style password-style"
+                                type="password"
+                            />
```

Comment:
There is a cool feature with labels in which you can click them, and the input field would highlight.

> <label for="password" ...>
> <input id="password" ...>

You can tell a label which is its input field when you use the key "for" having in mind that what is inside is the ID of the input field.

Reply:
Completely forgot to implement this. At first we had put the input fields inside the label, in this case, clicking the label would focus the input field. But styling caused us to separate the input field from the label tags and forgot to add the _for_ attribute. Thanks for spotting this!